core/spec/mailers/shipment_mailer_spec.rb

```diff
@@ -90,4 +90,18 @@
       end
     end
   end
+
+  context 'emails contain only urls of the store where the order was made' do
+    before do
```

Comment:
@aplegatt do we need this before block? I've tried running test without it locally and it passed. The same happened for order and reimbursement mailer specs